### PR TITLE
Allow theme colors in Row block placeholder

### PR DESCRIPTION
**Pull request by paranoia1906**
I have two commits on this branch and am looking for design feedback.

Commit `2f15c96` accomplishes a similar look to WP 5.3. This route give the placeholder a typical look in the editor when viewing in 5.2.4. 

**WP 5.3**
https://prnt.sc/pmpsyk

**WP 5.2.4**
https://prnt.sc/pmpt84

Commit `9b8962b` is using `theme(primary)` to set SVG and border colors for button. This route accomplishes a similar look to WP 5.3  but effects 5.2.4 as well. 

**How it looks in 5.3 and 5.2.4**
https://prnt.sc/pmpsit

### CODE REVIEW

**Comment by paranoia1906** [comment]
closes #1051

**Comment by richtabor** [comment]
I'd like feedback from the others as well, but I think it's fine to merge the styles prior to 5.3 landing.

File: src/components/visual-dropdown/styles/editor.scss [hunk 1]
```diff
@@ -186,29 +169,13 @@
 	width: 63px;
 }
 
-.components-coblocks-visual-dropdown__popover {
```
Comment (richtabor) [inline src/components/visual-dropdown/styles/editor.scss]: Let's make sure this doesn't affect the Map block's styles dropdown (if you have a Google Map API Key)

Comment (paranoia1906) [reply src/components/visual-dropdown/styles/editor.scss]: It was indeed effected. I had to adjust lines 175-181 to get the styles to look correct but that is now adjusted. Looks right in 5.2.4 and 5.3. Looks good with and without GB running.

**Pull request merged**